sandy/formats/mf33.py

```diff
@@ -30,12 +30,12 @@ def read(text):
             if subsub["LTY"] == 0:
                 L, i = read_list(str_list, i)
                 subsub.update({"E1" : L.C1, "E2" : L.C2,
-                               "CI" : L.B[:L.N2], "XMTI" : L.B[L.N2:]})
+                               "CI" : L.B[::2], "XMTI" : L.B[1::2]})
```

Comment:
is this correct? why are you looking at MF33? IF you change something here I would:

- [ ]  do it on a different branch,
- [ ] add a test

also,

Reply:
inconsistency with Master branch...keep the option coming from master